README.md

```diff
@@ -59,6 +59,43 @@ Setting `mask`, as done for the send options above, will cause the data to be
 masked according to the WebSocket protocol. The same option applies for text
 data.
 
+### Sending iovecs
```

Comment:
I agree. any suggestions?
I could only come up with a super explicit sentence -
 "Sending a message composed of multiple buffers".